genesyscloud/resource_genesyscloud_init.go

```diff
@@ -54,7 +54,7 @@ func registerDataSources(l registrar.Registrar) {
 	l.RegisterResource("genesyscloud_routing_wrapupcode", ResourceRoutingWrapupCode())
 
 	l.RegisterDataSource("genesyscloud_user", DataSourceUser())
-	l.RegisterDataSource("genesyscloud_telephony_providers_edges_extension_pool", dataSourceExtensionPool())
+	//l.RegisterDataSource("genesyscloud_telephony_providers_edges_extension_pool", dataSourceExtensionPool())
```

Comment:
If these are not needed you can remove them

Reply:
addressed